styles/sass/forms.scss

```diff
@@ -1009,6 +1009,115 @@ textarea {
     }
 }
 
+// boxed inputs
+input, textarea, .dropdown__toggle {
```

Comment:
Would it not work if boxed-input class wasn't restricted to `input, textarea, .dropdown__toggle`?

Reply:
not at the moment, because of some overrides. it has to stay this way for now. it can be cleaned up during some general refactoring of all fields in the header.